pkg/setting/setting.go

```diff
@@ -1377,7 +1378,12 @@ func (cfg *Cfg) readFeatureToggles(iniFile *ini.File) error {
 
 func readAlertingSettings(iniFile *ini.File) error {
 	alerting := iniFile.Section("alerting")
-	AlertingEnabled = alerting.Key("enabled").MustBool(true)
+	enabled, err := alerting.Key("enabled").Bool()
+	if err == nil {
+		AlertingEnabled = &enabled
+	} else {
```

Comment:
I don't think we need the `else` here as `AlertingEnabled` has a default value of `nil`?

Reply:
In my opinion, this helps readability.